src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -281,7 +296,7 @@ public void afterNavigateTo(String url, WebDriver driver) {
       try {
         new JavascriptActions(driver).execute("$(\".sprite.close-notification\")[0].click()");
       } catch (WebDriverException e) {
-
+        //TODO: Handle exception
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)